nw/gui/build.py

```diff
@@ -331,16 +331,24 @@ def __init__(self, theParent, theProject):
         self.fileGroup.setLayout(self.fileForm)
 
         self.novelFiles = QSwitch(width=wS, height=hS)
-        self.novelFiles.setToolTip(self.tr(
-            "Include files with layouts 'Book', 'Page', 'Partition', "
-            "'Chapter', 'Unnumbered', and 'Scene'."
+        self.novelFiles.setToolTip(self.tr("Include files with layouts {0}.").format(
+            self.locale().createSeparatedList([
+                self.locale().quoteString(self.tr('Book')),
+                self.locale().quoteString(self.tr('Page')),
+                self.locale().quoteString(self.tr('Partition')),
+                self.locale().quoteString(self.tr('Chapter')),
+                self.locale().quoteString(self.tr('Unnumbered')),
+                self.locale().quoteString(self.tr('Scene')),
+            ])
```

Comment:
@vkbo This is how I suggest working with lists and quoted terms.

Reply:
I think this is too much to be honest. Translation already creates overhead, and this is just a tooltip.